boa3/analyser/moduleanalyser.py

```diff
@@ -588,6 +588,8 @@ def visit_Subscript(self, subscript: ast.Subscript) -> Union[str, IType]:
         """
         value = self.visit(subscript.value)
         symbol = self.get_symbol(value) if isinstance(value, str) else value
+        # if isinstance(symbol, IExpression):
+        #     symbol = symbol.type
```

Comment:
We shouldn't have 'dead code'. If this removal is permanent, we need to delete it. If it's not, we need to add a todo and create an issue to track it.